src/uiConfig/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {Index} from '../pages/Index';
+import {First} from '../pages/First';
+import {Alternate} from '../pages/Alternate';
+import { Test } from '../pages/Test';
+
+
+// All component which will be rendered
+export const componentsSet : any = {
+    Index,
+    First,
+    Alternate,
+    Test
```

Comment:
We shall try to have different components here (than the routes/pages/screens ones) @subhamtsd 
Try to create simple ListEntities, ShowEntity, AddEditEntity (with Form pulled in)

That will help beginners to avoid getting confused, and stressing the fact that pages and components are different

Reply:
Hi @saurshaz ,

Ok sure from tomorrow I will start my work on that from Monday morning. 
For AddEditEntity Can I use the Json Form schema itself (2 to 3 text fields) just for the sake of Demo ?

Reply:
Hi @saurshaz ,

Ok sure from tomorrow I will start my work on that from Monday morning.
For AddEditEntity Can I use the Json Form schema itself (2 to 3 text fields) just for the sake of Demo ?

Reply:
yes .pls use those